setup.py

```diff
@@ -19,6 +19,16 @@
 ]
 
 
+class bdist_wheel_abi3(bdist_wheel):
```

Comment:
this is inspired from https://github.com/joerick/python-abi3-package-sample/blob/main/setup.py, which is the repo that cibuildwheel's docs cites as a reference implementation.
This class is used in replacement of the static metadata from `setup.cfg`